code/world.dm

```diff
@@ -11,7 +11,7 @@
 	cache_lifespan = 7
 	hub = "Exadv1.spacestation13"
 	icon_size = WORLD_ICON_SIZE
-	fps = 20
+	fps = 50
```

Comment:
Are you sure about this? Is this not going to force the ticker to run way faster now?
Also, do you have any reason for this specific number, or is it just taken from somewhere like /tg/

Reply:
We use 50 fps but it wasn't hard defined